definitions/ext-router/definition.yml

```diff
@@ -0,0 +1,26 @@
+domain: EXT
+# Network Router devices from Kentik using Metric API
+# Leaving this ambiguous to allow future technical partners to use the same entity type with additional conditions
+type: ROUTER
+synthesis:
+  name: device_name
+  identifier: device_name
+  encodeIdentifierInGUID: true
+
+  conditions:
```

Comment:
we'll only really apply 1 of these conditions atm (the telemetry won't have to match both), right @srvaroa?

Reply:
this telemetry's current design will require matching both; if we can't do that please let me know ASAP so I can go back to the vendor and request a change. thanks!!!

Reply:
Thaks for the change!